tests/test_freqtradebot.py

```diff
@@ -3545,8 +3545,34 @@ def test_get_real_amount_no_trade(default_conf, buy_order_fee, caplog, mocker, f
                    caplog)
 
 
-def test_get_real_amount_stake(default_conf, trades_for_order, buy_order_fee, fee, mocker):
-    trades_for_order[0]['fee']['currency'] = 'ETH'
+@pytest.mark.parametrize(
+    'fee_cost, fee_currency, fee_reduction_amount, use_ticker_rate, expected_log',
+    [
+        (None, 'ETH', 0, True, None),
+        (0.004, None, 0, True, None),
+        (0.00094518, "BNB", 0, True, None),
+        (
+            0.004,
+            "LTC",
+            0.004,
+            False,
+            (
+                'Applying fee on amount for Trade(id=None, pair=LTC/ETH, amount=8.00000000, '
```

Comment:
The log assertions seem like they could be calculated, would that be preferred over asserting a static value?